packages/blocks/README.md

```diff
@@ -354,7 +354,19 @@ Returns all the block categories.
 
 _Returns_
 
--   `Array<Object>`: Block categories.
+-   `Array<WPBlockTypeCategory>`: Block categories.
+
+<a name="getCategory" href="#getCategory">#</a> **getCategory**
```

Comment:
Should probably be consistent here with `getDefaultCategory`, if it's kept in the implementation. There was some prior concern about including this at all, since these singleton methods are discouraged. In any case, they should either _both_ be included, or _both_ be omitted, depending if the current implementation stays.